action.yml

```diff
@@ -24,6 +24,10 @@ inputs:
     description: "Version of troposphere to install"
     required: false
     default: "2.7.1"
+  sceptre_plugins:
```

Comment:
could we add some docs for this?  also not all sceptre plugins are in pypi.  Can we pip install directly from github?

Reply:
I added an example to the docs, and I confirmed support for GitHub repos. See my comment below.